### DISCUSSION TITLE
Adding an image in read_index.html is not working

### DISCUSSION THREAD

**Discussion by Martayugui** [bluenviron/mediamtx#3632] [Questions]
### Question

Hi everybody,

I am customizing the web page via read_index.html file. One of the new features I would like to add is to display a image in the middle of the screen when video streams are not coming.
I have run the same test with another html file and it has worked.

I have added the image using some JS code, CSS and HTML:

#header img {

    position: fixed;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    width: 20%;

}

`<div id="header" class="center"></div>`

                var img = document.createElement("img");
                img.src = "photo.svg";
                var src = document.getElementById("header");
                src.appendChild(img);
                
                
What I am getting:
![image](https://github.com/user-attachments/assets/363c5de7-cbe7-4eb8-97ee-5d8f73c90ace)

I have tried different ways to show the image but any of them have worked.
Any idea?

Thank you in advance!

**Comment by aler9** [comment]
duplicate of #3633
